app/services/schools/register_ect.rb

```diff
@@ -112,6 +114,19 @@ def update_school_last_choices!
       school.update!(**choices.compact)
     end
 
+    def close_ongoing_ect_period!
+      return unless teacher
+
+      ongoing_period = teacher.ect_at_school_periods.ongoing.started_before(started_on).first
+      return unless ongoing_period
```

Comment:
I think we can make this more concise using a `has_one`, like the [has one ongoing_induction_period](https://github.com/DFE-Digital/register-early-career-teachers-public/blob/main/app/models/teacher.rb#L20) relationship.

For now though let's get this merged and then increment.